src/hooks/usePolicyWithConnections.ts

```diff
@@ -0,0 +1,25 @@
+import {useEffect} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import {openPolicyAccountingPage} from '@libs/actions/PolicyConnections';
+import ONYXKEYS from '@src/ONYXKEYS';
+import useNetwork from './useNetwork';
+
+export default function usePolicyConnections(policyID: string) {
```

Comment:
NAB: Wondering if we can use this hook inside the existing HOC to make the code dry.

Reply:
yep I'll do that as a cleanup issue once release is done 😄